### PR TITLE
Fixed exclude parameter for files & folders

### PR DESCRIPTION
**Pull request by fwolfsjaeger**
This should fix issue #48

### CODE REVIEW

**Comment by wapmorgan** [comment]
Have you tested it?

**Comment by fwolfsjaeger** [comment]
> Have you tested it?

Sure, I've attached the output of the test run.
[phpcf.log](https://github.com/wapmorgan/PhpCodeFixer/files/3931893/phpcf.log)

File: src/PhpCodeFixer.php [hunk 1]
```diff
@@ -141,13 +141,25 @@ public function checkDir($dir) {
...
+
         foreach (glob($dir.'/*') as $file) {
+            $path_info = pathinfo($file);
+
+            if (
+                in_array(strtolower($path_info['filename']), $this->excludeList, true)
+                || in_array(strtolower($path_info['basename']), $this->excludeList, true)
+            ) {
+                $location_type = (is_dir($file)) ? 'Folder' : 'File';
+                $report->addInfo(Report::INFO_MESSAGE, $location_type . ' ' . $file . ' skipped');
```
Comment (wapmorgan) [inline src/PhpCodeFixer.php]: I think you missed here a `continue` statement.

File: src/PhpCodeFixer.php [hunk 2]
```diff
@@ -141,13 +141,25 @@ public function checkDir($dir) {
      * @param Report $report
      */
     protected function checkDirInternal($dir, Report $report) {
+        if (in_array(strtolower(basename($dir)), $this->excludeList, true)) {
```
Comment (wapmorgan) [inline src/PhpCodeFixer.php]: I think that's unnecessary condition

Comment (fwolfsjaeger) [reply src/PhpCodeFixer.php]: Without that one you can't exclude top level folders. You can test it by excluding vendor or src within this repository.

Comment (fwolfsjaeger) [reply src/PhpCodeFixer.php]: It's missing the continue statement as well though.

File: src/PhpCodeFixer.php (hunk 1 above)
Comment (fwolfsjaeger) [reply src/PhpCodeFixer.php]: Absolutely correct, the continue statement is missing.

**Comment by fwolfsjaeger** [comment]
If there are any issues with my changes just let me, I don't want to cause you additional work.

**Comment by wapmorgan** [comment]
No, everything is good. Thanks for contribution

**Pull request merged**